.circleci/config.yml

```diff
@@ -30,60 +34,99 @@ common_filters: &common_filters
       ignore:
         - gh-pages
 
+common_jobs: &common_jobs
+    jobs:
+        - find-develop-snapshot
```

Comment:
This is included in `common_jobs`, but inside `find-develop-snapshot`, it's only triggered when `condition: << pipeline.parameters.run_against_develop_core >>`. You could move this out of `common_jobs` and into `nightly` since the `everything` workflow doesn't use it. It would remove this "empty" job for `everything` https://app.circleci.com/pipelines/github/dockstore/dockstore-cli/1714/workflows/535c449e-f348-4ff2-b46a-6af475c0b7fa/jobs/8503

Reply:
> This is included in `common_jobs`, but inside `find-develop-snapshot`, it's only triggered when `condition: << pipeline.parameters.run_against_develop_core >>`. You could move this out of `common_jobs` and into `nightly` since the `everything` workflow doesn't use it. It would remove this "empty" job for `everything` https://app.circleci.com/pipelines/github/dockstore/dockstore-cli/1714/workflows/535c449e-f348-4ff2-b46a-6af475c0b7fa/jobs/8503

I tried to address this, but you [can't share parts](https://stackoverflow.com/questions/9254178/is-there-yaml-syntax-for-sharing-part-of-a-list-or-map) of a YAML list via YAML anchors. In other words, I couldn't have the `find-develop-snapshot` job as part of the `nightly` build, then add the rest of the jobs via the YAML alias.

So there are two choices:

1. Leave it the way it is, with the empty job being "run" every time for regular builds, or
2. Get rid of `common_jobs` and mainly duplicate the list of jobs for both everything and nightly in the config.yml.

I don't really like either one of these approaches, but lean towards no duplication being the lesser evil.

@kathy-t, who raised the issue, and anybody else monitoring this, are you OK with that? I know you already approved it, but just wanted to make sure.

Reply:
> I don't really like either one of these approaches, but lean towards no duplication being the lesser evil. 

Agreed, looking superficially we get charged per minute and there doesn't seem to be a cost for container startup

Reply:
I'm okay with keeping it the way you have in the PR. Could you add a comment about it? I feel like I would forget about why it's that way in the future and accidentally go down the route of trying to remove it :grin: